### DISCUSSION TITLE
Working on issue #12033 Negative float/double constants fail.

### DISCUSSION THREAD

**Discussion by ludollaoo** [micropython/micropython#16134] [Core Development]
Hi everyone,
I am looking to contribute and I came across this issue : https://github.com/micropython/micropython/issues/12033  Negative float/double constants fail. I just wanted to check if someone is already working on it. Thanks!

**Comment by aliarifat** [comment]
As far as I can understand, no one has currently been assigned to resolve MicroPython issue #12033.

**Reply by ludollaoo** [reply]
I got in touch with some contributors on discord. They confirmed me that no one was assigned to this issue but also told me to start with a simpler one : https://github.com/micropython/micropython/issues/11284 f-string thousands separator doesn't work with floats, so I will start with the latter.
